system/lib/standalone/standalone.c

```diff
@@ -109,15 +202,247 @@ __attribute__((__weak__)) int __syscall_fcntl64(int fd, int cmd, ...) {
   return -ENOSYS;
 }
 
+void to_public_stat(const __wasi_filestat_t *in,
```

Comment:
make this `static`?

Reply:
Its not clear to me what `public` means here.   How about `from_wasi_stat`?  Or `covert_wasi_stat`?